spec/src/main/asciidoc/ch03-entity-operations.adoc

```diff
@@ -1958,31 +1950,29 @@ to the returned set of `ConstraintViolation` objects, and must mark the
 transaction for rollback if the persistence context is joined to the
 transaction.
 
-The validator instance that is used for
-automatic validation upon lifecycle events must use a
-`TraversableResolver` that has the following behavior:
-
-* Attributes that have not been loaded must not
-be loaded.
-* Validation cascade (`@Valid`) must not
-occur for entity associations (single-valued or multi-valued).
+The validation process must not have any side effects;
+i.e., validation must not result in the loading of any
+attribute or association which is not already loaded.
+An entity can be validated at most once per flush cycle.
 
-These requirements guarantee that no unloaded
-attribute or association will be loaded by side effect and that no
-entity will be validated more than once during a given flush cycle.
+To achieve this, persistence provider must initialize
+the validator instance in use with the traversable resolver
+(`jakarta.validation.TraversableResolver`)
+that matches these behavior requirements, in particular:
 
-Embeddable attributes must be validated only
-if the `Valid` annotation has been specified on them.
+* `TraversableResolver#isReachable(..)` only returns `true`
+for already initialized attributes and associations.
+* `TraversableResolver#isCascadable(..)` only returns `true`
+for properties that are not associations.
```

Comment:
The term "loaded" is used in the passage above this list.
Also, please prefer the syntax of Java to the syntax of Javadoc.

```suggestion
* `TraversableResolver.isReachable(..)` only returns `true`
  for attributes and associations which are already loaded.
* `TraversableResolver.isCascadable(..)` only returns `true`
  for attributes that are not associations.
```